scalafmt-sbt/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -15,23 +15,32 @@ object ScalafmtPlugin extends AutoPlugin {
   override def trigger: PluginTrigger = allRequirements
 
   object autoImport {
-    val scalafmt = taskKey[Unit]("Format Scala sources")
+    val scalafmt = taskKey[Unit]("Format Scala sources.")
     val scalafmtCheck =
-      taskKey[Boolean]("Check that Scala sources is formatted properly")
+      taskKey[Boolean](
+        "Fails if a Scala source is mis-formatted. Does not write to files.")
     val scalafmtOnCompile =
-      settingKey[Boolean]("Format source when compiling")
-    val scalafmtConfig = taskKey[File]("Scalafmt config file")
-    val scalafmtSbt = taskKey[Unit]("Format SBT sources")
+      settingKey[Boolean](
+        "Format Scala source files on compile, off by default.")
+    val scalafmtConfig = taskKey[Option[File]](
```

Comment:
Does this mean that you must `scalafmtConfig := Some(file(".scalafmt.conf"))` to avoid the default settings? I would prefer to make this non-Option and if the file does not exist then use default settings.

Reply:
It searches for `.scalafmt.conf` file by default. And you can redefine it for your custom file.

Reply:
I see, cool. OK this is fine then 👍